### ISSUE TITLE
Cannot fully download some ubuntu files

### ISSUE THREAD

**Issue by gamesover** [Ansuel/tch-nginx-gui#907]
NB: Before submitting an issue, check if there is one already open that suits the problem you are having!

<b>If you are on the latest dev, please go to Cards >> System Extras >> Debug Report and press the Generate button.
  Please attach the file here:</b>
[DebugHelper2020-03-20-1505.tar.gz](https://github.com/Ansuel/tch-nginx-gui/files/4357959/DebugHelper2020-03-20-1505.tar.gz)

Device Model: MediaAccess TG ( MediaAccess TG789MYRvac v2 HP )
Gui Version: 9.5.17-5b723682

Description of problem:
Some ubuntu files cannot be downloaded when I use `sudo apt install xxx`. such as http://au.archive.ubuntu.com/ubuntu/pool/main/g/gcc-7/cpp-7_7.5.0-3ubuntu1~18.04_amd64.deb. When I use chrome to download directly, it is frozen at 3.8/8.2 MB
How to reproduce (if possible):
When I use  TG789MYRvac v2 HP with custom GUI, it always has this strange problem. Reboot, no improve.

However, when switching to a diff regular modem, belong modem 4315, or use mobile internet, I can download http://au.archive.ubuntu.com/ubuntu/pool/main/g/gcc-7/cpp-7_7.5.0-3ubuntu1~18.04_amd64.deb without any issue.
Media/Photos (if possible):
![Screenshot from 2020-03-21 22-04-06](https://user-images.githubusercontent.com/3777577/77225607-72139400-6bc0-11ea-9a7b-3039ad077003.png)

**Comment by FrancYescO** [comment]
more tha a tch-nginx-gui issue looks like you are having ISP connectivity issue.. 
are you connected in VDSL? did you have changed the VDSL driver? on lan transfers are working good?

**Comment by gamesover** [comment]
I am with NBN. Everything is good except cannot fully download some ubuntu packages. But if I switch to another regular modem, no such issue. Only TG789MYRvac v2 modem has this strange issue. It may be due to TG789 software or hardware issue.

**Comment by FrancYescO** [comment]
Try changing MTU, what's the connection type used pppoe?

**Comment by gamesover** [comment]
How can I change MTU? My isp auto assign me a dynamic ip, no need to use username/password to login. 

![Screenshot from 2020-03-21 21-12-16](https://user-images.githubusercontent.com/3777577/77224879-c5361880-6bb9-11ea-8321-cc1ed0c54bf8.png)

**Comment by FrancYescO** [comment]
Si IPoE? DHCP over Wan?

/etc/config/network

wan block for the MTU change

**Comment by gamesover** [comment]
My is DHCP over Wan and wan block of `/etc/config/network`
```
config interface 'wan'           
        option vci '35'             
        option vpi '8'                               
        option ifname 'eth4'      
        option reqopts '1 3 6 15 33 42 43 51 121 249'
        option release '1'                           
        option vendorid 'dslforum.org'
        option sendopts '61:01E0B9E5D67E16'
        option ipv6 '0'                              
        option peerdns '1'    
        option proto 'dhcp'           
        option mtu '1500'   
```
Originally, there is no `option mtu`, I added and changed to diff value 1500, 1460, 1492, 1400.
No diff, still cannot fully download http://au.archive.ubuntu.com/ubuntu/pool/main/g/gcc-7/cpp-7_7.5.0-3ubuntu1~18.04_amd64.deb.

And I open https://www.speedguide.net/analyzer.php to test, it is said my optimized mtu should be 1500.

**Comment by FrancYescO** [comment]
sincerely i have no clue whats happening here and how the GUI can cause this, do you still have the issue?
try disabling Ethernet Optimization or something other about the cpu in eco functions, flow access control or if present on your device DOS protect.. also try to check if putting firewall in low protection mode helps

**Comment by gamesover** [comment]
Hello @FrancYescO , I really appreicate for your always great helps. Now, I switched to my old telstra modem v7610. Everything is back to normal. I am quite happy with it.

For now, I am not going to debug TG789 since if any issue happens, it is very easy to brick the modem. I've bricked several modems already ;)

When you release next big version v10, I will try to upgrade this TG789 to see whether it could solve the issue.

Really thanks for your help.
